src/styles/stepTracker.css

```diff
@@ -1,6 +1,7 @@
 .step-tracker-container {
   display: flex;
   flex-direction: row;
+  position: relative;
```

Comment:
What is position relative needed for?

Reply:
I thought you put absolute and relative on things when layering?
https://www.w3schools.com/css/css_positioning.asp